code/modules/effects/map_effects/marker/large_tank.dm

```diff
@@ -0,0 +1,58 @@
+
+/// Large tank marker, used to set up "large tanks", that being multi-tile tanks of gas,
+/// with its own control console, sensor, pump, and injector.
+/obj/effect/map_effect/marker/large_tank
+	name = "large tank marker"
+	desc = "See comments/documentation in code."
+	icon = 'icons/effects/map_effects.dmi'
+	icon_state = "marker_large_tank"
+	layer = LIGHTING_LAYER
+
+	/// Radio frequency of this large tank.
+	var/frequency = 2199
+
+	/// Unique tag for this large tank. Not visible in game and to the player. Do not leave this as null.
+	/// THIS MUST BE UNIQUE FOR THE LARGE TANK. Every marker in one large tank should have the same `master_tag`.
+	/// Different large tanks, even on different maps, cannot share the same `master_tag`.
+	var/master_tag = null
```

Comment:
Need a way to verify that this is enforced, otherwise it would be too easy to break ships/sites from completely different ships/sites in way that look random and would be a pain to identify during debugging

Reply:
We do not enforce unique ids/tags for anything else so I see no reason to enforce it here.
We do not enforce map-unique ids/tags for shutters, buttons, shuttles, shuttle landmarks, or anything else like that.

This would not be easy at all to enforce either, without having nasty global lists.